byte[] dxl_ids
int16[] torque_limits
int16[] present_speeds
int16[] present_loads
byte[] present_voltages
byte[] present_temperatures
byte[] error_bits
